.github/workflows/ci.yaml

```diff
@@ -141,6 +141,9 @@ jobs:
     name: E2E Tests
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    env:
```

Comment:
Use `NODE_OPTIONS="--max-old-space-size=2048" JOBS=max yarn --cwd ui build` in `ui` directly instead of defining ENVs.